src/app/server/Server.cpp

```diff
@@ -237,7 +231,35 @@ CHIP_ERROR Server::Init(AppDelegate * delegate, uint16_t secureServicePort, uint
     SuccessOrExit(err);
 
     err = mCASESessionManager.Init();
-
+#if !CHIP_DEVICE_CONFIG_ENABLE_THREAD
+    {
+        ChipLogProgress(AppServer, "Adding Multicast groups");
+        ConstFabricIterator fabricIterator = mFabrics.cbegin();
+        while (!fabricIterator.IsAtEnd())
+        {
+            FabricInfo fabric = *fabricIterator;
```

Comment:
This this need to be a copy?  Or can this be a const reference?

Reply:
done